src/general-components/PaginationFooter/PaginationFooter.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react';
+import {PageItem, Pagination} from 'react-bootstrap';
+
+import './paginationFooter.scss'
+
+interface PaginationFooterProps {
+    /**
+     * Anzahl der Seiten, welche zur Auswahl angezeigt werden sollen
+     */
+    pageCount: number
+    /**
+     * Welche Seite gerade ausgewählt ist
+     *
+     * startet bei 1
+     */
+    currentPage: number
+    /**
+     * callback, wenn eine Seite ausgewählt wurde
+     * @param pageNumber nummer der ausgewählten Seite (startet bei 1)
+     */
+    pageChosen: (pageNumber: number) => void
+
+    /**
+     * legt fest ob eingaben möglich sein sollen
+     */
+    disabled?: boolean
+}
+
+/**
+ *
+ */
+class PaginationFooter extends Component<PaginationFooterProps, {}> {
+
+
+    static defaultProps = {
+        currentPage: 1
+    };
+
+    itemClicked(pageNumber: number, e: React.MouseEvent) {
+        e.preventDefault();
+        if (pageNumber !== this.props.currentPage)
+            this.props.pageChosen(pageNumber);
+    }
+
+    nextClicked = (e: React.MouseEvent) => {
+        e.preventDefault();
+        this.props.pageChosen(this.props.currentPage + 1)
+    };
+
+    previousClicked = (e: React.MouseEvent) => {
+        e.preventDefault();
+        this.props.pageChosen(this.props.currentPage - 1)
+    };
+
+    render() {
+                    // clamp active value between 1 and this.props.pageCount
```

Comment:
spacing is weird

Reply:
was chosen like that to specify this comment is related to the if statement. I am removing the indentation